content/post/057-ipfs-camp-course-videos.md

```diff
@@ -0,0 +1,80 @@
+
+This workshop introduced a few awesome visualization and learning tools that you can try out at home:
+
+- Explore the [ProtoSchool tutorial on the Mutable File System (MFS)](https://proto.school/#/mutable-file-system) for coding challenges designed to familiarize you with its API
+- Paste a Content Identifier (CID) into the [CID Inspector](https://cid.ipfs.io/) to break down its anatomy
+- Drag a file into the [DAG Builder](https://dag.ipfs.io/) and adjust chunk size, DAG layouts, etc. to see how it affects the Merkle DAG
+
+Watch the recording below or check out the [slides](https://github.com/ipfs/camp/blob/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A/IPFS_Camp_Core_Course_A_Slides.pdf)!
+
+[![Video: Understanding How IPFS Deals with Files](/img/057-ipfs-camp-course-videos/core-a-thumbnail.png)](https://youtu.be/dJ7TkB4UpZ8)
+
+_Video: [Understanding How IPFS Deals with Files](https://youtu.be/dJ7TkB4UpZ8)_
+
+
+### The Lifecycle of Data in DWeb
+
+This course covers all the stuff that happens _after_ you add a file to IPFS, highlighting the intricacies of how peers interact with each other to provide, get, pin, and delete blocks of data. It explains the Distributed Hash Table (DHT) that helps us track which peers are storing which CIDs, as well as how Bitswap requests and sends blocks while verifying their data. No poodle pics here, but the live Bitswap dance using bean bags to model the block-sharing process is not to be missed! 🤹‍♂️
+
+Watch the video below or check out the slides (available on [Google Slides with speaker notes](https://docs.google.com/presentation/d/1UOX-a1lCxi-LOyEOe9VEFRcxg_j93Xkywwa4Y95TJaw/edit?usp=sharing) or as a [PDF](https://github.com/ipfs/camp/blob/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_D/IPFS_Camp_Core_Course_D_Slides.pdf))!
+
+[![Video: The Lifecycle of Data in DWeb](/img/057-ipfs-camp-course-videos/core-d-thumbnail.png)](https://youtu.be/fLUq0RkiTBA)
+
+_Video: [The Lifecycle of Data in DWeb](https://youtu.be/fLUq0RkiTBA)_
+
+
+
+### And more!
+
+There are plenty more core and elective courses to explore in the latest batch of videos. Check out our full [course playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtSsHMhjeWpfOzr8tonPaePu) or jump right to the video that looks most interesting:
+
+**Core Courses**
+
+- [Understanding how the InterPlanetary File System deals with files](https://youtu.be/dJ7TkB4UpZ8)
+- [Solving distributed networking problems with libp2p](https://youtu.be/Q4IH5rWEO-E)
+- [Developing apps with IPFS API](https://youtu.be/6EHBOnFQJN4)
+- [The lifecycle of data in DWeb](https://youtu.be/fLUq0RkiTBA)
+
+**Elective Courses**
```

Comment:
should we say who was involved in each of these courses? in most cases it's clear (cluster, textile, qri, etc), but the infra one in particular should give credit to @obo20 for the great presentation!

Reply:
Thanks for the suggestion @momack2. Are you thinking that we should list all of the credited teammates for each preso (as found in the repo and the video title slides)? I'm under the impression that some were very much a team effort as indicated there and some ended up much more on one person to create, in which cases I wouldn't know who that was for most of these. 🤷‍♀️

Reply:
Just pushed an edit that adds all of the authors listed in the camp repo for each course.